gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb

```diff
@@ -0,0 +1,596 @@
+        "- The content in the JSON instructions must match the structure of a [GenerateContentRequest](https://cloud.devsite.corp.google.com/vertex-ai/generative-ai/docs/model-reference/inference#request_body).\n",
+        "- The input table can have columns other than `request`. They are ignored for content generation but included in the output table. The system reserves two column names for output: `response` and `status`. These are used to provide information about the outcome of the batch prediction job.\n",
+        "- Only public YouTube and Cloud Storage bucket URIs in the `fileData` or `file_data` field are supported in batch prediction.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3d4b751610f0"
+      },
+      "source": [
+        "This is an example BigQuery table:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "uWb8QzxwbH6W"
+      },
+      "outputs": [],
+      "source": [
+        "INPUT_DATA = \"bq://storage-samples.generative_ai.batch_requests_for_multimodal_input\"  # @param {type:\"string\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b7aeba876320"
+      },
+      "source": [
+        "You can query the BigQuery table to review the input data."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "1414f03a7999"
+      },
+      "outputs": [],
+      "source": [
+        "bq_client = bigquery.Client(project=PROJECT_ID)\n",
+        "\n",
+        "bq_table_id = INPUT_DATA.replace(\"bq://\", \"\")\n",
+        "sql = f\"\"\"\n",
+        "        SELECT *\n",
+        "        FROM {bq_table_id}\n",
+        "        \"\"\"\n",
+        "\n",
+        "query_result = bq_client.query(sql)\n",
+        "\n",
+        "df = query_result.result().to_dataframe()\n",
+        "df.head()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "T3jQ59mCsXLc"
+      },
+      "source": [
+        "### Prepare batch output location\n",
+        "\n",
+        "When a batch prediction task completes, the output is stored in the location that you specified in your request.\n",
+        "\n",
+        "- The location is in the form of a Cloud Storage or BigQuery URI prefix, for example:\n",
+        "`gs://path/to/output/data` or `bq://projectId.bqDatasetId`.\n",
+        "\n",
+        "- If not specified, `STAGING_BUCKET/gen-ai-batch-prediction` will be used for Cloud Storage source and `bq://PROJECT_ID.gen_ai_batch_prediction.predictions_TIMESTAMP` will be used for BigQuery source.\n",
+        "\n",
+        "This tutorial uses a **BigQuery** table as an example.\n",
+        "\n",
+        "- You can specify the URI of your BigQuery table in `BQ_OUTPUT_URI`, or\n",
```

Comment:
Can you add a specification that the notebook has code to create the dataset? When I first read it, I thought you meant that the API service itself creates a dataset if not provided.

Reply:
Updated. Thanks!